app/graphql/types/mutation_type.rb

```diff
@@ -3,5 +3,6 @@
 module Types
   class MutationType < Types::BaseObject
     field :create_election, mutation: Mutations::CreateElectionMutation
+    field :key_ceremony, mutation: Mutations::KeyCeremonyMutation
```

Comment:
I have a suggestion 😄 . As a good practice it's better to name mutations as actions so I suggest something like `process_key_ceremony_step` or something like that. What do you think?

Reply:
❤️ I thought exactly the same, but I didn't spend more time searching for a better name. Actually, the mutation was originally called KeyCeremonyStep, but I've removed the Step because it didn't fix the action thing. I'll use your proposal, that I like it too 👍

Reply:
I'll also change the related job and command with the same name. What do you think about that?

Reply:
Sorry for the late response! Yeah I also think that having naming consistency is necessary so I'd rather rename the job and the command as well 😄